### PR TITLE
Fixing expected parameter format as this must be a non-string parameter

### PR DESCRIPTION
**Pull request by makentenza**
Fixing template 'scc/project-run-anyuid-template.yml' for a non-string parameter required which is translated right now as a string.

### CODE REVIEW

File: scc/project-run-anyuid-template.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ objects:
   fsGroup:
     type: RunAsAny
   apiVersion: v1
-  priority: "${PRIORITY_LEVEL}"
+  priority: "${{PRIORITY_LEVEL}}"
```
Comment (etsauer) [inline scc/project-run-anyuid-template.yml]: Are you sure you want to wrap a non-string parameter in quotes?

**Review by etsauer — COMMENTED** [review]
@makentenza one question

File: scc/project-run-anyuid-template.yml (hunk 1 above)
Comment (makentenza) [reply scc/project-run-anyuid-template.yml]: We could change that if you think it makes more sense but one the template is processed it doesn't make any difference as the value appears as an 'int':

```
mak 🎩︎ @ MakF ~/GIT/openshift-templates/scc (fix/scc-template)
└─ ▶ grep priority: project-run-anyuid-template.yml
  priority: "${{PRIORITY_LEVEL}}"

mak 🎩︎ @ MakF ~/GIT/openshift-templates/scc (fix/scc-template)
└─ ▶ oc process -f project-run-anyuid-template.yml NAME=dummy NAMESPACE=dummy|grep priority
            "priority": 8,
```

Comment (pabrahamsson) [reply scc/project-run-anyuid-template.yml]: It works either way but my argument would be to remove the quotes just to clarify/emphasize that this is an integer. I can go either way, just let me know @makentenza and I will approve and merge.

Comment (makentenza) [reply scc/project-run-anyuid-template.yml]: @pabrahamsson @etsauer it's fine by me, just removed quotes for this parameter.

**Review by pabrahamsson — APPROVED** [review]
Thank you @makentenza, LGTM

**Pull request merged**